### ISSUE TITLE
Unrecognized option stimeout when starting SRS stream

### ISSUE THREAD

**Issue by johnatank** [tropicoo/hikvision-camera-bot#48]
Hi,
When starting srs stream, it failed to start due to bad stimeout option:
```
hikvision_camera_bot  | 2022-09-02 14:44:06,780 - [INFO] - [asyncio:1637] - run shell command 'ffmpeg -loglevel error -reorder_queue_size 1000000 -buffer_size 1000000  -rtsp_transport tcp -stimeout 10000000 -i "rtsp://xxx:yyy!@n.n.n.n:554/Streaming/Channels/101/"  -c:v copy  -c:a aac  -ar 22050 -f flv -flvflags no_duration_filesize -rtmp_live live "rtmp://hikvision_srs_server/live/livestream_101_cam_1"': <_UnixSubprocessTransport pid=20 running stdout=<_UnixReadPipeTransport fd=12 polling>>
hikvision_camera_bot  | 2022-09-02 14:44:06,782 - [INFO] - [FfmpegStdoutReaderTask:21] - Starting FfmpegStdoutReaderTask
hikvision_camera_bot  | 2022-09-02 14:44:06,783 - [DEBUG] - [FfmpegStdoutReaderTask:23] - Reading stdout from ffmpeg -loglevel error -reorder_queue_size 1000000 -buffer_size 1000000  -rtsp_transport tcp -stimeout 10000000 -i "rtsp://xxx:yyy!@n.n.n.n:554/Streaming/Channels/101/"  -c:v copy  -c:a aac  -ar 22050 -f flv -flvflags no_duration_filesize -rtmp_live live "rtmp://hikvision_srs_server/live/livestream_101_cam_1"
hikvision_camera_bot  | 2022-09-02 14:44:06,808 - [INFO] - [FfmpegStdoutReaderTask:24] - Unrecognized option 'stimeout'.
```

stimeout is an option for ffmpeg defined in hikcamerabot/constants.py:
```
FFMPEG_CMD_SRS = (
    f'{_FFMPEG_BIN} {_FFMPEG_LOG_LEVEL} '
    '-reorder_queue_size 1000000 '
    '-buffer_size 1000000 '
    '{filter} '
    '-rtsp_transport {rtsp_transport_type} '
    '-stimeout 10000000 '
    f'-i {FFMPEG_CAM_VIDEO_SRC} '
    '{map} '
    '-c:v {vcodec} '
    '{{inner_args}} '
    '-c:a {acodec} {abitrate} {asample_rate} '
    '-f {format} '
    '-flvflags no_duration_filesize '
    '-rtmp_live live '
    '"{{output}}"'
)
```

It appears option [was changed in ffmpeg 5+](https://stackoverflow.com/questions/70679738/unrecognized-option-stimeout). Option name should be timeout now. Modifying hikcamerabot/constants.py and rebuilding image resolves the error.

ffmpeg 5 is used in camera bot image:
```
docker exec hikvision_camera_bot ffmpeg -v
ffmpeg version 5.0.1 Copyright (c) 2000-2022 the FFmpeg developers
```

Thanks,
Maksym

**Comment by tropicoo** [comment]
Hey, thanks for bringing this up!

**Comment by tropicoo** [comment]
Fixed in [1.5](https://github.com/tropicoo/hikvision-camera-bot/releases/tag/1.5).
